external/include/mbedtls/see_api.h

```diff
@@ -27,23 +27,23 @@
 
 #include <stdio.h>
 
-#include "../../arch/arm/src/s5j/sss/mb_cmd_dh.h"
-#include "../../arch/arm/src/s5j/sss/mb_cmd_hash.h"
-#include "../../arch/arm/src/s5j/sss/mb_cmd_rsa.h"
-
-#include "../../arch/arm/src/s5j/sss/isp_define.h"
-#include "../../arch/arm/src/s5j/sss/isp_driver_hash.h"
-#include "../../arch/arm/src/s5j/sss/isp_driver_rng.h"
-#include "../../arch/arm/src/s5j/sss/isp_driver_secure_storage_factorykey.h"
-#include "../../arch/arm/src/s5j/sss/isp_driver_secure_storage.h"
-#include "../../arch/arm/src/s5j/sss/isp_driver_secure_storage_key.h"
-#include "../../arch/arm/src/s5j/sss/isp_driver_dh_securekey.h"
-#include "../../arch/arm/src/s5j/sss/isp_driver_rsa_securekey.h"
-#include "../../arch/arm/src/s5j/sss/isp_driver_hmac_securekey.h"
-#include "../../arch/arm/src/s5j/sss/isp_driver_ecdsa_securekey.h"
-#include "../../arch/arm/src/s5j/sss/mb_cmd_secure_storage_data.h"
-#include "../../arch/arm/src/s5j/sss/isp_driver_error.h"
-#include "../../arch/arm/src/s5j/sss/isp_oid.h"
+#include "mb_cmd_dh.h"
```

Comment:
If other module includes this header, it will not find them. Do we add an including path thru CFLAGS at every module which try to include this header?

Reply:
2. They are headers of s5j secure element. Is this file only for s5j?
  If it isn't, we need to bind a conditional.